### PR TITLE
Add an api which provides a reverse proxy for s3 files.

### PR DESCRIPTION
**Pull request by alan-wu**
# Description

An API for getting data from s3 bucket.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested on an internal instance.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -129,6 +129,20 @@ def create_presigned_url(expiration=3600):
 
     return response
 
+# Reverse proxy for objects from S3, a simple get object
+# operation. This is used by scaffoldvuer and its 
+# important to keep the relative <path> for accessing
+# other required files.
+@app.route("/object-resource/<path:path>")
```
Comment (hsorby) [inline app/main.py:136]: I would probably have the route called `s3-resource`

File: app/main.py [hunk 2]
```diff
@@ -129,6 +129,20 @@ def create_presigned_url(expiration=3600):
...
+def download_download_url(path):
```
Comment (hsorby) [inline app/main.py:137]: I would probably call this simply `download_url`.

File: app/main.py (hunk 1 above)
Comment (alan-wu) [reply app/main.py:136]: Ok.

File: app/main.py (hunk 2 above)
Comment (alan-wu) [reply app/main.py:137]: I meant to put down direct_download_url there... hmmm...

**Pull request merged**